### PR TITLE
CHI-1260: Deploy all prod workflow

### PR DESCRIPTION
**Pull request by stephenhand**
## Description

* Add a generic 'deploy multiple helplines for multiple environments workflow'
* Refactor 'Deploy All Staging Accounts' workflow to use above workflow
* Add a 'Deploy All Production Accounts' workflow using the same workflow
* Updated deploy workflow to only need the environment specifying once
* Updated single helpline deploy to use multiple deploy as well

### Checklist
- [X] Corresponding issue has been opened

### Related Issues
CHI-1260

### Verification steps

* Test the new 'deploy all prod' workflow & regression test the 'deploy all staging' and single helpline deploy workflows (particularly dev)

### CODE REVIEW

**Comment by stephenhand** [comment]
@robert-bo-davis @GPaoloni FYI

File: .github/workflows/config/production-helplines.json
Comment (murilovmachado) [inline .github/workflows/config/production-helplines.json:3]: You have only `AS` here for testing, but eventually, we'll list them all, correct?

**Review by murilovmachado — APPROVED** [review]
Looks good to me. I'm going to borrow the same idea for the `flex-plugins` repo

File: .github/workflows/config/production-helplines.json
Comment (stephenhand) [reply .github/workflows/config/production-helplines.json:3]: Yes, sorry, should have explained in the PR it's the old chicken & egg problem, shouldn't merge the PR before it's tested but can't test the new workflow without it being merged

Comment (stephenhand) [reply .github/workflows/config/production-helplines.json:3]: Yes, sorry should have noted that on the PR

The old 'chicken & egg' problem that we shouldn't merge before we test but the new workflow can't be tested before it's merged

**Pull request merged**
